packages/libs/react-ui/src/components/Table/index.ts

```diff
@@ -0,0 +1,25 @@
+import { ITableProps, Table as TableContainer } from './Table';
+import { ITBodyProps, TBody } from './TBody';
+import { ITdProps, Td } from './Td';
+import { IThProps, Th } from './Th';
+import { ITHeadProps, THead } from './THead';
+import { ITrProps, Tr } from './Tr';
+
+import { FC } from 'react';
+
+export { ITableProps, ITBodyProps, ITHeadProps, ITrProps, IThProps, ITdProps };
+
+interface ITable extends FC<ITableProps> {
+  Body: FC<ITBodyProps>;
+  Head: FC<ITHeadProps>;
+  Tr: FC<ITrProps>;
+  Th: FC<IThProps>;
+  Td: FC<ITdProps>;
+}
+
+export const Table: ITable = TableContainer as ITable;
```

Comment:
Can you achieve this without a type cast?

Reply:
As we discussed, replaced it with a container variable, exporting the `<Table>` component as `Table.Root`